packages/vitepress-plugin-og-image-generator/src/index.ts

```diff
@@ -0,0 +1,100 @@
+import { fileURLToPath } from 'node:url'
+import { dirname, relative, resolve, sep } from 'node:path'
+import fs from 'fs-extra'
+import fg from 'fast-glob'
+import { Resvg } from '@resvg/resvg-js'
+import type { Config } from './types/config'
+import type { ArticleTree } from './types/metadata'
+import { removeEmoji } from './utils'
+
+export default async function run(config: Config) {
+  const { sidebar, plainTargetDomain, dist } = config
+  const DIR_ROOT = resolve(dirname(fileURLToPath(import.meta.url)), '..')
+
+  // TODO: Load public files in a better way
+  const ogSvg = fs.readFileSync(resolve(DIR_ROOT, './public/og-template.svg'), 'utf-8')
+
+  let articles: ArticleTree[] = ([] as any).concat(...sidebar.map(series => [...series?.items.map(item => ({ ...item, category: series.text }))] || []))
```

Comment:
This line is kind hard to read too.

```suggestion
  let articles: ArticleTree[] = []
  
  articles = articles.concat(...sidebar.map((series) => {
    return [...series?.items.map((item) => {
      retrun { 
        ...item,
        category: series.text
      }
    })] || []
  }))
```

Reply:
How about this?